README.md

```diff
@@ -195,16 +195,42 @@ In this tool we have 2 dashboards, you can access them on `Dashboard` section
 
 ## Deployment
 
-Configuring and deploying to remote node instances is supported through Ansible playbooks. The default settings deploy to three node instances via SSH which host all layers of your metagraph project (gL0, mL0, cL1, dL1). Two hydra methods are available to help with the deployment process: `hydra remote_configure` and `hydra remote_deploy`.
+Configuring, deploying, and starting remote node instances is supported through Ansible playbooks. The default settings deploy to three node instances via SSH which host all layers of your metagraph project (gL0, mL0, cL1, dL1). Two hydra methods are available to help with the deployment process: `hydra remote_deploy` and `hydra remote_start`.
 Prior to running these methods, remote host information must be configured in  `infra/ansible/hosts.ansible.yml`
 
-### `hydra remote_configure`
 
-This method configures remote instances with all the necessary dependencies to run a Metagraph, including Java, Scala, and required build tools. The Ansible playbook used for this process can be found and edited in `infra/ansible/playbooks/deploy.ansible.yml`. 
+### Host Configuration
 
-### `hydra remote_deploy`
+To run your metagraph remotely, you'll need remote hosts—ideally, at least three of them. These hosts should be running either `ubuntu-20.04` or `ubuntu-22.04`. It's recommended that each host meets the following minimum requirements:
+
+-   16GB of RAM
+-   8vCPU
+-   160GB of storage
+
+You can choose your preferred platform for hosting your instances, such as AWS or DigitalOcean. After creating your hosts, you'll need to provide the following information in the `hosts.ansible.yml` file:
+
+-   Host IP
+-   Host user
+-   Host SSH key (optional if your localhost already has access to the remote host)
+
+### P12 Files
+
+These files serve as your token files and should be located in the `source/p12-files` directory by default. Details such as the `file-name`, `key-alias`, and `password` should be specified in the `euclid.json` file under the `p12_files` section. By default, Euclid comes with three example files: `token-key.p12`, `token-key-1.p12`, and `token-key-2.p12`. **NOTE:** Before deploying, be sure to replace these example files with your own, as these files are public and their credentials are shared.
+
+Each `p12` file contains a `peerId`, and each node is associated with a `peerId`. **NOTE:** Ensure that your peerIDs are registered on the MainNet network. Otherwise, the metagraph startup will fail because the network will reject the snapshots.
```

Comment:
```suggestion
**NOTE:** If deploying to MainNet, ensure that your peerIDs are registered and present on the metagraph seedlist. Otherwise, the metagraph startup will fail because the network will reject the snapshots.
```